src/UserBundle/Tests/Repository/UserInvitationRepositoryTest.php

```
@@ -93,4 +93,32 @@ public function testGetGridQuery(): void
         $alias = $queryBuilder->getRootAliases()[0];
         self::assertCount(1, $queryBuilder->getDQLPart('select'));
     }
+
+    public function testCountPendingScopesByCompany(): void
+    {
+        $executor = $this->databaseTool->loadFixtures([LoadData::class], true);
+        $inviter = $executor->getReferenceRepository()->getReference('user2', User::class);
+
+        $registry = self::getContainer()->get('doctrine');
+        $em = $registry->getManager();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$em'. (undefined)

(UnusedLocalVariable)